mesonbuild/interpreter/interpreter.py

```diff
@@ -1776,10 +1785,8 @@ def _func_custom_target_impl(self, node, args, kwargs):
             # string in the meantime.
             FeatureNew('custom_target() with no name argument', '0.60.0', location=node).use(self.subproject)
             name = ''
-        kwargs['install_mode'] = self._get_kwarg_install_mode(kwargs)
         if 'input' in kwargs:
             kwargs['input'] = self.source_strings_to_files(extract_as_list(kwargs, 'input'), strict=False)
-        kwargs['env'] = self.unpack_env_kwarg(kwargs)
```

Comment:
Hmm, I'm guessing these could be removed independent of the typed_kwargs conversion? Neither is supported by vcs_tag, so they would always end up as None, and any other entry point into `_func_custom_target_impl` either has them as None also, or converts them correctly?